src/test/java/easytests/api/v1/controllers/QuestionsControllerTest.java

```diff
@@ -14,11 +88,79 @@
     /**
      * update
      */
+
+    @Test
+    public void testShowWithAnswersSuccess() throws Exception {
+        final QuestionModelInterface questionModel = new QuestionModel();
+        questionModel.map(this.questionSupport.getEntityFixtureMock(0));
+        final List<AnswerModelInterface> answersModels = new ArrayList<>();
+        IntStream.range(0, 2).forEach(idx -> {
+            final AnswerModel answerModel = new AnswerModel();
+            answerModel.map(this.answersSupport.getEntityFixtureMock(idx));
+            answersModels.add(answerModel);
+        });
+        questionModel.setAnswers(answersModels);
+        when(this.questionsOptionsBuilder.forAuth()).thenReturn(new QuestionsOptions());
```

Comment:
take your attention on that mock, now it is not necessary. But when you begin use forAuth you will need to modify it. Think of "how".